test/org/traccar/reports/ReportUtilsTest.java

```diff
@@ -184,18 +179,18 @@ public void testDetectStopsStartedFromTrip() throws ParseException {
                 position("2016-01-01 00:04:00.000", 0, 0),
                 position("2016-01-01 00:05:00.000", 0, 0));
 
-        TripsConfig tripsConfig = new TripsConfig(500, 300000, 200000, false, 900000, false);
+        TripsConfig tripsConfig = new TripsConfig(500, 300000, 200000, 900000, false);
 
-        Collection<? extends BaseReport> result = ReportUtils.detectTripsAndStops(tripsConfig, false, 0.01, data, false);
+        Collection<StopReport> result = ReportUtils.detectStops(data, tripsConfig, false, 0.01);
 
         assertNotNull(result);
         assertFalse(result.isEmpty());
 
-        StopReport itemStop = (StopReport) result.iterator().next();
+        StopReport itemStop = result.iterator().next();
 
-        assertEquals(date("2016-01-01 00:00:00.000"), itemStop.getStartTime());
+        assertEquals(date("2016-01-01 00:02:00.000"), itemStop.getStartTime());
```

Comment:
The second difference: If period started from moving it will not be added to detected parking (more logically I think)